jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -130,20 +133,26 @@ private static EventHandlers makeEventHandlers(
     }
     ConsoleLogger consoleLogger = logEventHandlerBuilder.build();
 
-    return new EventHandlers()
-        .add(
-            JibEventType.LOGGING,
-            logEvent -> consoleLogger.log(logEvent.getLevel(), logEvent.getMessage()))
-        .add(
-            JibEventType.TIMING,
-            new TimerEventHandler(message -> consoleLogger.log(LogEvent.Level.DEBUG, message)))
-        .add(
-            JibEventType.PROGRESS,
-            new ProgressEventHandler(
-                update ->
-                    consoleLogger.setFooter(
-                        ProgressDisplayGenerator.generateProgressDisplay(
-                            update.getProgress(), update.getUnfinishedAllocations()))));
+    EventHandlers eventHandlers =
+        new EventHandlers()
+            .add(
+                JibEventType.LOGGING,
+                logEvent -> consoleLogger.log(logEvent.getLevel(), logEvent.getMessage()))
+            .add(
+                JibEventType.TIMING,
+                new TimerEventHandler(message -> consoleLogger.log(LogEvent.Level.DEBUG, message)));
+
+    if (session.getRequest().isInteractiveMode()) {
```

Comment:
We should still add the dumb plain progress messages (just a few lines of message when not using the fancy progress bars).

```java
    ConsoleLoggerBuilder logEventHandlerBuilder =
        (isProgressFooterEnabled()
                ? ConsoleLoggerBuilder.rich(singleThreadedExecutor)
                : ConsoleLoggerBuilder.plain(singleThreadedExecutor).progress(log::info))
```

So, the check should be done inside `isProgressFooterEnabled()`. Will need passing `session` to the method.